### PR TITLE
Adds cssClass prop for wrapper custom styling

### PR DESCRIPTION
**Pull request by obayomi96**
- Adds a new prop called `cssClass` for custom styling

## Issue Fixed
- #137

### CODE REVIEW

**Comment by femioladeji** [comment]
Cool @obayomi96. Can you add a test to ensure that the cssClass exists on that element when the prop is passed

File: __tests__/fade.test.js [hunk 1]
```diff
@@ -218,3 +218,14 @@ test('When the pauseOnHover prop is true and the mouse hovers the container the
...
+
+test('cssClass prop exists on element when it is passed', () => {
+  const { container } = renderFade({
+    ...fadeProperties,
+    cssClass: 'myStyle'
+  });
+  let elementToStyle = container.querySelectorAll(
+    '.react-slideshow-fade-wrapper'
+  );
+  expect(elementToStyle.length).toBe(1);
```
Comment (femioladeji) [inline __tests__/fade.test.js:230]: This assertion doesn't actually check if `myStyle` was added to the `.react-slideshow-fade-wrapper`
You should use something like this 
```js
const wrapper = container.querySelectorAll(
    '.react-slideshow-fade-wrapper'
  );
  expect(wrapper.classList).toContain('myStyle');
```

**Review by femioladeji — CHANGES_REQUESTED** [review]
Only put the test in the test file marked as 2

File: __tests__/fade.test.js (hunk 1 above)
Comment (obayomi96) [reply __tests__/fade.test.js:230]: Thanks!, Will update it.

**Review by femioladeji — APPROVED** [review]

**Pull request merged**